rackcity/views/it_instance_views.py

```diff
@@ -167,3 +174,15 @@ def instance_page_count(request):
     instance_count = ITInstance.objects.all().count()
     page_count = math.ceil(instance_count / page_size)
     return JsonResponse({"page_count": page_count})
+
+
+def is_location_full(rack_id, elevation, height):
+    rack_u = [elevation + i for i in range(height)]
+    instances_in_rack = ITInstance.objects.filter(rack=rack_id)
+    for instance in instances_in_rack:
+        for u in [
```

Comment:
Same comment for `u`. Maybe `occupied_range`?

Reply:
Actually I guess `occupied_slot` or `occupied_location` is more accurate

Reply:
excellent